persistent-sqlite/cbits/config.c

```diff
@@ -4,3 +4,7 @@
 int persistent_sqlite_set_log(void (*logFn)(void*, int, const char*), void* arg) {
   return sqlite3_config(SQLITE_CONFIG_LOG, logFn, arg);
 }
+
+long long int persistent_sqlite3_soft_heap_limit64(long long int N) {
```

Comment:
What is this function adding versus just calling `sqlite3_soft_heap_limit64` directly?